src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Xunit.Performance.Sdk;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal static class XunitBenchmark
```

Comment:
Yes, I think it makes sense as we are moving towards supporting broader scenarios